spec/classes/backend_tessen_spec.rb

```diff
@@ -0,0 +1,48 @@
+require 'spec_helper'
+
+describe 'sensu::backend::tessen', :type => :class do
+  on_supported_os({
+    facterversion: '3.8.0',
```

Comment:
Do we have other tests that are always the same for each OS that we are testing with each OS? Perhaps there is an opportunity to speed things up.

Reply:
The only place we currently change behavior for OS is the repo classes.  Everything else is not OS specific but not sure that will always be the case, especially when Windows support gets added.

Reply:
One way to speed up at least unit tests is to use parallel rspec which would execute each spec file as separate process.